setup.py

```diff
@@ -51,21 +51,25 @@ def run(self):
     license="Apache 2.0",
     description="Cloudstate Python Support Library",
     packages=packages,
+    package_data={
+        # If any package contains *.proto files, include them:
+        "": ["*.proto"],
+    },
     long_description=open("Description.md", "r").read(),
     long_description_content_type="text/markdown",
     zip_safe=False,
-    scripts=["scripts/compile-protobuf.sh", "scripts/fetch-cloudstate-pb.sh"],
+    scripts=["scripts/fetch-cloudstate-pb.sh"],
     install_requires=[
         "attrs>=19.3.0",
         "google-api>=0.1.12",
         "googleapis-common-protos >= 1.51.0",
-        "grpcio>=1.28.1",
-        "grpcio-tools>=1.28.1",
+        "grpcio>=1.31.0",
+        "grpcio-tools>=1.31.0",
         "protobuf>=3.11.3",
         "pytest>=5.4.2",
         "six>=1.14.0",
-        "grpcio-reflection>=1.28.1e",
-        "docker"
+        "grpcio-reflection>=1.31.0",
```

Comment:
1.32.x ?

Reply:
The code works at major release 1.31 and since we have the dependency version wild-carded, we should be able to install in envs where the user has 1.32.x. I can test that things work with 1.32 to verify this...

Reply:
Yeah, the tcks pass when we use 1.32